### PR TITLE
Fixed: Misalignment of placeholder text inside date component in the …

### LINKED ISSUE #8239: Misalignment of placeholder text inside date component in the add user page
**Describe the bug**

the Placeholder text in the date component in the add user page is not aligned when compared with the other Placeholder text in the same page

**To Reproduce**

1. Go to user tab
2. Click on add user button
3. See error

**Expected behavior**

all the Placeholder text in the form should be properly aligned and consistent

**Screenshots**

image

### PR DESCRIPTION
**Pull request by 0xvish**
Misalignment of placeholder text inside date component in the add user page #8239

## Proposed Changes

- Fixes #8239
- Changed the CSS to add padding on the left to align the placeholder in DateInputV2

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

Screenshot 2024-08-02 005250

### CODE REVIEW

File: src/Components/Common/DateInputV2.tsx [hunk 1]
```diff
@@ -256,7 +256,7 @@ const DateInputV2: React.FC<Props> = ({
                   type="text"
                   readOnly
                   disabled={disabled}
-                  className={`cui-input-base cursor-pointer !px-2 disabled:cursor-not-allowed ${className}`}
+                  className={`cui-input-base cursor-pointer !px-2 pl-4 disabled:cursor-not-allowed ${className}`}
```
Comment (rithviknishad) [inline src/Components/Common/DateInputV2.tsx]: `cui-input-base` class applies consistent padding and colors already. Removing `!px-2 pl-4` seems to have also solved the issue. Let's stick to not overriding the default padding applied for all input elements.

```suggestion
                  className={`cui-input-base cursor-pointer disabled:cursor-not-allowed ${className}`}
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Comment by nihal467** [comment]
@thevishvammoliya nice work, feel free to join the community slack workspace : https://join.slack.com/t/rebuildearth/shared_invite/zt-2blycdt83-yvoJRsa10QDhE9WFiwoVkg

**Comment by 0xvish** [comment]
@nihal467 thanks for all the support, looking forward to contribute more to CARE ❤️

**Pull request merged**